server/models/TransactionSettlement.ts

```diff
@@ -77,6 +77,7 @@ class TransactionSettlement extends Model<
         AND t."deletedAt" IS NULL
         AND ts."deletedAt" IS NULL
         ${settlementStatus ? 'AND ts."status" = :settlementStatus' : ''}
+        ORDER BY "id" ASC
```

Comment:
Not related to this PR, but the test somehow randomly fails because the transactions order was inconsistent

![image](https://user-images.githubusercontent.com/1556356/223430773-ddc93eaf-6194-4c57-87cf-5a72fd184989.png)